### PR TITLE
Prepare release 0.9.0

### PR DESCRIPTION
**Pull request by cehbrecht**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes issue #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [x] I have added my relevant user information to `AUTHORS.md`

* **What kind of change does this PR introduce?:** 

This PR prepares the next release of daops. 

* **Does this PR introduce a breaking change?:** 

requires clisops >=0.10 to support tests on Python 3.11.

* **Other information:**

### CODE REVIEW

**Comment by cehbrecht** [comment]
@Zeitsperre @alaniwi this is just a patch release ... but because I changed the minimal version of clisops to 0.10 I would make it a 0.9.0 release. Are you ok with this?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -12,21 +12,21 @@ repos:
       - id: check-yaml
       - id: debug-statements
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 23.1.0
```
Comment (Zeitsperre) [inline .pre-commit-config.yaml:15]:
```suggestion
    rev: 23.11.0
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -12,21 +12,21 @@ repos:
...
       - id: flake8
         args: ['--config=setup.cfg']
   - repo: https://github.com/asottile/reorder_python_imports
-    rev: v3.0.1
+    rev: v3.9.0
     hooks:
       - id: reorder-python-imports
   - repo: https://github.com/asottile/pyupgrade
-    rev: v2.32.0
+    rev: v3.3.1
```
Comment (Zeitsperre) [inline .pre-commit-config.yaml:29]:
```suggestion
    rev: v3.15.0
```

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -12,21 +12,21 @@ repos:
...
       - id: debug-statements
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 23.1.0
     hooks:
       - id: black
         args: ["--target-version", "py37"]
   - repo: https://github.com/pycqa/flake8
-    rev: 4.0.1
+    rev: 6.0.0
```
Comment (Zeitsperre) [inline .pre-commit-config.yaml:20]:
```suggestion
    rev: 6.1.0
```

File: HISTORY.rst [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+v0.9.0 (2023-11-15)
+-------------------
+
+Breaking Changes
+^^^^^^^^^^^^^^^^
+* ``clisops``>=0.10.0 required.
+
+Other Changes
+^^^^^^^^^^^^^
+
```
Comment (Zeitsperre) [inline HISTORY.rst:13]:
```suggestion
```

File: requirements_dev.txt [hunk 5]
```diff
@@ -3,7 +3,7 @@ bumpversion
 wheel
 watchdog
 flake8
-tox
+tox~=3.0
```
Comment (Zeitsperre) [inline requirements_dev.txt:6]: Why v3.0?

**Review by Zeitsperre — APPROVED** [review]

File: requirements_dev.txt (hunk 5 above)
Comment (cehbrecht) [reply requirements_dev.txt:6]: don't know ... probably from the cookiecutter template.

Comment (cehbrecht) [reply requirements_dev.txt]: updated.

**Pull request merged**